sirepo/auth/__init__.py

```diff
@@ -665,6 +665,14 @@ def _parse_display_name(value):
     return res
 
 
+def _plan(roles):
+    if 'enterprise' in roles:
```

Comment:
use constant ROLE_PREMIUM, change name to ROLE_PAYMENT_PLAN_PREMIUM and add ROLE_PAYMENT_PLAN_ENTERPRISE.

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/110ea9c3b63ca85561bac84b1dd582a383edab18